prime-router/build.gradle.kts

```diff
@@ -375,6 +375,13 @@ tasks.jar {
 tasks.shadowJar {
     // our fat jar is getting fat! Or over 65K files in this case
     isZip64 = true
+    // this package is tied to log4j1. it's coming in not as a dependency but somehow as source from one of the
+    // dependencies in the project. it's (for now) harmless but flotsam so removing it. The call to transform(...)
+    // addresses the bug.
```

Comment:
What bug? -- so this comment makes sense in the future

Reply:
I'm happy to change the comment if you'd like. I would also like to point out the link to the bug ticket is directly below. Is the link sufficient or do you want another sentence describing the bug as well?